generic/kdump.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Pavithra <pavrampu@linux.vnet.ibm.com>
+
```

Comment:
you can still improve the testcase by adding different kdump setup like:
sed -i \"/GRUB_CMDLINE_LINUX/s/crashkernel=auto/crashkernel=2G-4G:320M,4G-32G:512M,32G-64G:1024M,64G-128G:2048M,128G-:4096M/g\" /etc/default/grub
grub2-mkconfig -o /boot/grub2/grub.cfg

for different values.

Also add support for ubuntu kdump